h/search/query.py

```diff
@@ -89,6 +107,20 @@ def __call__(self, search, params):
                  "unmapped_type": "boolean"}}
         )
 
+    def _date_parser(self, str_value):
+        try:
+            return float(str_value)
+        except ValueError:
+            try:
+                date = parse(str_value, )
+                # If timezone isn't specified assume it's utc.
```

Comment:
Assuming UTC sounds like a good idea to me - since this will the values in the returned annotations.